bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -1580,11 +1580,11 @@ lnd/clm2/surfdata_esmf/NEON/surfdata_1x1_NEON_TOOL_hist_78pfts_CMIP6_simyr2000_c
 <flanduse_timeseries hgrid="C96"           sim_year_range="1850-2100" ssp_rcp="SSP2-4.5"
 >lnd/clm2/surfdata_esmf/ctsm5.2.0/landuse.timeseries_C96_SSP2-4.5_1850-2100_78pfts_c240216.nc</flanduse_timeseries>
 
-<flanduse_timeseries hgrid="ne0np4.ARCTICGRIS.ne30x8" sim_year_range="1979-2026" ssp_rcp="SSP2-4.5"
+<flanduse_timeseries hgrid="ne0np4.ARCTICGRIS.ne30x8" ssp_rcp="SSP2-4.5"
```

Comment:
@slevis-lmwg since this is a new situation, we should discuss how to handle this. I can think of quite a few different ways to do this, and still deciding what is the best way..

Reply:
I have now found a similar problem for the 1979 fsurdat files because IHist cases pick up the 1850 files by default. I have a temporary workaround for testing, but let's discuss options on Monday if you're available @ekluzek.

Reply:
Continuing this discussion [at the relevant line of code](https://github.com/ESCOMP/CTSM/pull/2490/commits/99d710015b83ebed8082a5ca0a980f317403bd77#r1585671993).